controllers/front/ProductController.php

```diff
@@ -815,6 +848,59 @@ public function getTemplateVarProduct()
         );
     }
 
+    /**
+     * @param $product
+     * @return int
+     */
+    protected function getProductMinimalQuantity($product)
+    {
+        $minimal_quantity = 1;
+
+        if ($product['id_product_attribute']) {
```

Comment:
This product attribute id comes from the request, I'd rather keep it as is.

See https://github.com/thierrymarianne/PrestaShop/blob/485ace4daa02edb740e22511727bc1def901aa0a/controllers/front/ProductController.php#L813